structure/0006-governance.md

```diff
@@ -2,53 +2,128 @@
+- **Prior Art**:
+  - Are there other organizations or projects with comparable tasks?
 
-If there are less than three members on the administrative board, each remaining board member may nominate a community member (potentially a non-staff member) for the position. Existing staff members may nominate themselves. The remaining staff members may then vote on the nominees, with administrative board member votes being worth three votes only in the event of a tie.
+### Sub Teams
 
-### Proposing a Team
-When proposing a new team, an RFC specifying several things needs to be written. Such a proposal should generally follow this structure:
+Some teams may wish for more granular organization of their structure.
+In those cases, these teams may create sub teams.
 
-#### Motivation
-Why is this team needed? What current problems will this team solve? What existing projects or systems will this team be responsible for?
+Sub teams behave like regular teams, except they are not required to have a team lead.
+If the team doesn't have its own team lead, the parent's team leads act as its team lead.
+In general, a sub team inherits its parent's structure according to their parent's team RFC.
 
-#### Explanation
-What will this team *do*? What new projects will be created that this team will be responsible for and how will each of these projects benefit the QuiltMC organization/community? What processes will this team follow for each of the different kinds of changes their projects may undergo?
+## The Administrative Board
 
-#### Drawbacks
-Can usually be omitted. Is only necessary if there are clear drawbacks to such a teams existence. Drawbacks may include overlap of responsibilities, excessive granularity, etc.
+The administrative board is a group of organization members responsible for tasks that are not assigned to another team.
+It is made up of at least three members to prevent deadlocks in the decision-making process.
+Members of the administrative board may be members of other teams.
 
-#### Rationale and Alternatives
-Can be omitted. Who is currently responsible for existing projects this team will takeover? What teams might become responsible for new projects proposed in this document if the team itself is deemed unnecessary?
+At any point in time, a member of the organization may nominate themselves for a position on the administrative board.
+This will then initiate the voting process, with all other organization members being eligible to vote.
+If the nominated member passes a two-thirds vote, they are added to the administrative board.
 
-#### Prior Art
-Can usually be omitted. Is only necessary if there are prior organizations/projects with very similar goals.
+If there are less than three members on the administrative board, a special vote is initiated.
```

Comment:
i'd rephrase "initiated" to "called" to fit in with the wording found under **voting**

Reply:
Yep, thanks. I changed the wording midway, I'll fix this.

Reply:
any voting surrounding the maintaining of the mandatory minimum of body should have an overal, deterministic deadline by which point (or earlier) new people are assigned to the body.

in my experience, if you leave the overal period indeterminate "as soon as possible", "when they have found a replacement", it can happen that the remaining member(s) drag their feet and it doesn't happen for months.

yes, the alternate election method is already included, but you will find that the majority will have a tendency to not hold this required vote and the few that want it are placated or leveraged to not hold the vote.

by making sure there is always a deadline, this will hopefully make sure the regular members are motivated to hold the vote, even if the remaining body member(s) have been dragging their feet on their required steps.